cmd/openshift/operator/kodata/tekton-addon/addons/01-clustertriggerbindings/github.yaml

```diff
@@ -39,6 +39,8 @@ spec:
     value: $(body.head_commit.message)
   - name: git-repo-url
     value: $(body.repository.url)
+  - name: git-repo-html-url
+    value: $(body.repository.html_url)
```

Comment:
Would it be clearer to use `git-repo-clone-url` or would that be overly specific?